packages/neuron-ui/src/components/BalanceSyncingIcon/index.tsx

```diff
@@ -10,13 +10,13 @@ export interface BalanceSyncIconProps {
 
 const BalanceSyncIcon = ({ connectionStatus, syncStatus }: BalanceSyncIconProps) => {
   if (ConnectionStatus.Connecting === connectionStatus || ConnectionStatus.Offline === connectionStatus) {
-    return <Confirming className={styles.confirm} />
+    return <Confirming className={`${styles.confirm} infinite-rotation`} />
```

Comment:
it can be modified to use the `clsx` function from the `utils` module.

Reply:
When will the `Confirming` not rotate infinitely?

Reply:
> When will the `Confirming` not rotate infinitely?

Add a property to control the status.